### PR TITLE
Optimize graphs 2

### PR DESCRIPTION
**Pull request by Norbiros**
Resolves #4183

### Changes

Create new option, hide fill and animation for smaller lags.

### Reason for changes

Some people have lags viewing graphs.

This is second attempt... First have some errors...

### CODE REVIEW

**Comment by Norbiros** [comment]
What's with it? I tested it 2 times... Can someone please help?
`I don't have this error in any other repositories, but in SA I have!` 
**I looked for all tutorials but I can't fix it...**

**Comment by CST1229** [comment]
> What's with it? I tested it 2 times... Can someone please help? `I don't have this error in any other repositories, but in SA I have!` **I looked for all tutorials but I can't fix it...**

What is the error?

**Comment by Norbiros** [comment]
@CST1229 https://github.com/ScratchAddons/ScratchAddons/runs/5079384164?check_suite_focus=true
With checks... Code is good for me.

**Comment by Norbiros** [comment]
Okay... I'll wait for @WorldLanguages to accept this. Thanks!

File: addons/debugger/performance.js [hunk 1]
```diff
@@ -4,7 +4,16 @@ export default async function createPerformanceTab({ debug, addon, console, msg
   const vm = addon.tab.traps.vm;
 
   await addon.tab.loadScript(addon.self.lib + "/thirdparty/cs/chart.min.js");
-
+  
+  // In optimized graphs it still looks good
+  if (addon.settings.get("graphs_optimization") == true) {
+      var lineWidth = 2;
```
Comment (apple502j) [inline addons/debugger/performance.js:10]: Please do not use `var`.

**Review by GrahamSH-LLK — COMMENTED** [review]
I really don't understand how a lot of these changes (lineColor, lineWidth) make rendering faster. Can you explain?

File: addons/debugger/performance.js [hunk 2]
```diff
@@ -4,7 +4,16 @@ export default async function createPerformanceTab({ debug, addon, console, msg
...
+      var lineColor = "hsla(163, 85%, 40%, 1)";
+  } else {
+      var lineWidth = 1;
+      var lineColor = "hsla(163, 85%, 40%, 0.5)";
+  }
```
[the remark this answers is not in the document]
Comment (Norbiros) [reply addons/debugger/performance.js:15]: Yes. When you have only default settings you can't see line in dark mode.

[the remark this answers is not in the document]
Comment (Norbiros) [reply addons/debugger/performance.js:15]: People with "write" permission - can you add this? Nice suggestion

[the remark this answers is not in the document]
Comment (GrahamSH-LLK) [reply addons/debugger/performance.js:15]: you have write access?! this is your PR. I'm an admin, but I don't have access to your branch

[the remark this answers is not in the document]
Comment (Norbiros) [reply addons/debugger/performance.js:15]: Me to... It looks like my branch doesn't exist. You can't event open it in my fork repository `Norbiros:optimize-graphs`. I tried to restore it but also i can't do it for this branch.

**Pull request closed without merging**